internal/command/commands/env.go

```diff
@@ -291,6 +292,51 @@ func (e *Env) Run(args []string) int {
 		return 1
 	}
 
+	if e.Update {
+
+		emptyParameters := emptyParameters(map[string]string{"--name": *name, "--file": *filePath})
+
+		if len(emptyParameters) == 0 {
+			err, parsedConfig := parseFile(*filePath)
+			if err != nil {
+				e.Logger.Error("Error while parsing service file, err: \n" + err.Error())
```

Comment:
You are not actually parsing service file

Reply:
we are not taking service file from the user to update an env. we are just parsing the file which has the details of the env to be updated